snippets/bootstrap-migration/product/price.liquid

```diff
@@ -0,0 +1,34 @@
+
+<dl class="d-flex justify-content-between mb-0">
+    <dt>
+        <span class="text-helvetica {% if variation_price %}font-weight-normal text-uppercase text-muted{% endif %}">
+            {% if customer %}
+                {{ 'products.price.private' | t }}
+            {% else %}
+                {{ 'products.price.public' | t }}
+            {% endif %}
+        </span>
+    </dt>
+
+    <dd class="font-weight-bold">
+        <span class="{% if variation_price %}{% else %}text-dark-green{% endif %}">
+            {% if variant != null %}
+                {% assign productOrVariant = variant %}
+            {% else %}
+                {% assign productOrVariant = product %}
+            {% endif %}
+            {% if settings.show_prices_with_taxes %}
+                {% assign price = productOrVariant.price_with_tax %}
```

Comment:
Fix formating please

Reply:
I there's nothing to fix in this file

Reply:
Agree, probably I was confused by `assign` and `include`